mysite/chat/views.py

```diff
@@ -319,7 +318,7 @@ def get_enroll_code_object(enroll_key):
         :param enroll_key: enroll code
         :return: EnrollUnitCode instance
         """
-        return get_object_or_404(EnrollUnitCode, enrollCode=enroll_key, isLive=True)
+        return get_object_or_404(EnrollUnitCode, enrollCode=enroll_key, isLive=True, chat__is_preview=False)
```

Comment:
that's correct change. I added field directly to EnrollUnitCode, because chat may not exist yet